README.md

```diff
@@ -3,277 +3,155 @@
-  datatoken for each dataset / data service.
-- **Mint** datatokens for the service
-- **Sell** datatokens via an OCEAN-datatoken Balancer pool (for auto price discovery), or for a fixed price
-- **Stake** OCEAN on datatoken pools
-- **Consume** datatokens, to access the service
-- **Transfer** datatokens to another owner, and **all other ERC20 actions**
-  using [web3.js](https://web3js.readthedocs.io/en/v1.2.9/web3-eth-contract.html) etc. -->
-
 Ocean v4 is part of the [Ocean Protocol](https://oceanprotocol.com) toolset.
 
-This is in alpha state and you can expect running into problems. If you run into them, please open up a [new issue](https://github.com/oceanprotocol/ocean.js/issues/new?assignees=&labels=bug&template=bug_report.md&title=).
+This is in beta state and you can expect running into problems. If you run into them, please open up a [new issue](https://github.com/oceanprotocol/contracts/issues/new?assignees=&labels=bug&template=bug_report.md&title=).
 
-- [📚 Prerequisites](#-prerequisites)
-- [Overview](#-overview)
+- [📚 Installation](#-installation)
 - [🏄 Quickstart](#-quickstart)
-  - [Features](#features)
+  - [What's New](#new-features)
   - [Publisher Flow](#publisher-flow)
   - [Roles Diagram](#roles-diagram)
   - [Functions you will need](#functions-you-will-need)
-  <!-- - [v3 Integration and support](#v3-integration-and-support) -->
-  
-- [🦑 Development](#-development)
-- [👩‍🔬 Testing](#-testing)
-  - [Unit Tests](#unit-tests)
-  - [Flow Tests](#flow-tests)
+  - [Bundle functions](#bundle-functions)
+- [🦑 Development and testing](#-development-testing)
 - [Known issues](#known-issues)
 - [🏛 License](#-license)
 
-## 📚 Prerequisites
-
-- node.js ([Install from here](https://nodejs.org/en/download/))
-<!-- - A Unix based operating system (Mac or Linux) -->
-
-<!-- ### Note
-
-Any function that uses `getPastEvents()` will only work on Eth (see: <https://github.com/oceanprotocol/ocean.js/issues/741>). This includes:
-
-- searchPoolforDT()
-- getPoolsbyCreator()
-- getPoolSharesByAddress()
-- getAllPoolLogs()
-- getPreviousValidOrders()
-- searchforDT()
-- getExchangesbyCreator()
-- getExchangeSwaps()
-- getAllExchangesSwaps() -->
+## 📚 Installation
 
-<!-- ## 🏗 Installation
+For quick installation of the contract `ABIs`:
 
+### Javascript/Typescript
 ```bash
-npm install @oceanprotocol/lib
-``` -->
+npm install @oceanprotocol/contracts
+```
+### Python
+```bash
+pip3 install ocean-contracts
+```
+#### ⚙️ Usage
+By default, Python does not support importing `json` files directly, so it is recommended to use `json-sempai` package in order to automatically importing `ABIs/json` artifacts.
+```
+pip3 install json-sempai
+# install the ocean-contracts package.
+```
+```python
+from jsonsempai import magic
+from artifacts import address
+```
+
 
 ## Overview
 
 ![image](docs/images/smart-contracts.png)
 
 ## 🏄 Quickstart
 
+### New Features 
```

Comment:
There should be no "New Features". It should simply list the key features, that's it. (This documentation is for our outside users, not internal users.)